src/Init/Init_FFTW.cpp

```diff
@@ -194,33 +192,33 @@ void End_FFTW()
    if ( MPI_Rank == 0 )    Aux_Message( stdout, "%s ... ", __FUNCTION__ );
 
 
-   destroy_real_fftw_plan   ( FFTW_Plan_PS      );
+   destroy_real_fftw_plan  ( FFTW_Plan_PS      );
 
 #  ifdef GRAVITY
-   destroy_real_fftw_plan   ( FFTW_Plan_Poi     );
-   destroy_real_fftw_plan   ( FFTW_Plan_Poi_Inv );
-#  endif
+   destroy_real_fftw_plan  ( FFTW_Plan_Poi     );
+   destroy_real_fftw_plan  ( FFTW_Plan_Poi_Inv );
+#  endif // #  ifdef GRAVITY
+
 
 #  if ( MODEL == ELBDM )
-   destroy_complex_fftw_plan( FFTW_Plan_Psi     );
-   destroy_complex_fftw_plan( FFTW_Plan_Psi_Inv );
-#  endif
+   destroy_complex_fftw_plan  ( FFTW_Plan_Psi     );
+   destroy_complex_fftw_plan  ( FFTW_Plan_Psi_Inv );
+#  endif // #if ( MODEL == ELBDM )
 
 #  if ( SUPPORT_FFTW == FFTW3 )
 #  ifdef OPENMP
    if (FFTW3_Double_OMP_Enabled)  fftw_cleanup_threads();
    if (FFTW3_Single_OMP_Enabled) fftwf_cleanup_threads();
-#  endif
-#  ifdef SERIAL
+#  endif // # ifdef OPENMP
+
    fftw_cleanup();
    fftwf_cleanup();
```

Comment:
From the [manual](https://www.fftw.org/fftw3_doc/MPI-Initialization.html), `fftw_mpi_cleanup()` calls `fftw_cleanup()`. So what's the purpose of this change?

If calling `fftw_cleanup()` even when disabling `SERIAL` is preferred, we should also update the `main` branch.

Reply:
@hyschive 
Thanks for catching this. Again, I did not read the documentation properly and was unsure whether `fftw_mpi_cleanup `would also call `fftw_cleanup`. When calling both worked fine, I just left it like that, but as the documentation states `fftw_mpi_cleanup` already calls `fftw_cleanup` so I just reverted it to what it was before.